uber/site_sections/preregistration.py

```diff
@@ -453,6 +453,9 @@ def transfer_badge(self, session, message='', **params):
     def confirm(self, session, message='', return_to='confirm', **params):
         attendee = session.attendee(params, bools=_checkboxes, restricted=True)
 
```

Comment:
As a stylistic matter, I'd probably be inclined to make ``invalid_badge`` its own page and then do a ``raise HTTPRedirect`` to it.  In this case there's little to no difference, but that would make it marginally easier to add parameters and such.